services/pokemon_service.go

```diff
@@ -0,0 +1,36 @@
+package services
+
+import (
+	"golangBootcamp/m/common"
+	"golangBootcamp/m/models"
+	"strconv"
+)
+
+func FindAllPokemons() ([]models.Pokemon, error) {
+	csvPokemons, error := common.ReadCsvFile("./pokemon.csv")
+	if error != nil {
+		return nil, error
+	}
+	pokemons := []models.Pokemon{}
+	for _, csvPokemon := range csvPokemons {
+		intId, error := strconv.Atoi(csvPokemon[0])
```

Comment:
I don't really see doing this parsing on the service side as a bad thing... But I would probably leave the parsing of the rows to the package responsible of dealing with csv files, and let it be like a "bridge" between csv rows and the data structures we use on the api.

This thought can also be applied to packages that, for example, call an external API or databases. We usually don't want to deal with raw http responses or db-like structures. So we let the app layers that communicate with other resources to parse the data/response/etc... into a data structure that is more specific.

Doing this would mean that your package that reads csv files would have a function that could be called `GetPokemonsFromCSV`